src/components/modals/ExportModal.vue

```diff
@@ -62,11 +65,16 @@ export default class ExportModal extends Vue {
     console.log(`Files to be exported: ${this.selectedFiles}`);
     console.log(`Should export Overview?: ${this.overviewSelected === CheckboxValue.CHECKED}`);
 
-    // TODO: Should actually use selected stuff
-    let generatedCode = '';
-    const { name, state } = saveEditor(this.currEditor);
-    const graph = istateToGraph(state);
-    if (this.currEditorType === EditorType.OVERVIEW) {
+    const zip = new JSZip();
+
+    const modelsFolder = zip.folder('models');
+    const dataFolder = zip.folder('data');
+    const codevaultFolder = zip.folder('codevault');
+
+    // generate overview
+    if (this.overviewSelected) {
```

Comment:
So we don't want to export if we're in another editor? I'm not sure I understand this bit here.